libs/service-portal/settings/access-control/src/screens/GrantAccess/GrantAccess.tsx

```diff
@@ -64,13 +73,11 @@ function GrantAccess() {
   const [getIdentity, { data, loading: queryLoading }] = useLazyQuery<Query>(
     IdentityQuery,
     {
-      onError: (error) => {
-        toast.error(
-          formatMessage({
-            id: 'service.portal.settings.accessControl:grant-identity-error',
-            defaultMessage: 'Enginn notandi fannst með þessa kennitölu.',
-          }),
-        )
+      onError: noUserFoundToast,
```

Comment:
Ideally "errors" are handled separately in some globally consistent way. But that's outside the scope of this PR.

Reply:
Yeah true, but it's related to my project's link: https://github.com/orgs/island-is/projects/9#card-60800557

Currently there's no error handling for the service portal 🙈 
All errors are "swallowed" and just console logged.